bitrise.yml

```diff
@@ -68,10 +67,44 @@ workflows:
             cp $DEPLOY_PATH $BITRISE_DEPLOY_DIR/$BIN_NAME-$OS-$ARCH
             echo "  Copy final Darwin binary to: $BITRISE_DEPLOY_DIR/$BIN_NAME-$OS-$ARCH"
 
+            # Create M1 bin
+            export ORIG_DIR=`pwd`
+            export GO_TMP=/tmp/go1.16.4 && mkdir -p $GO_TMP
+              
+            UNAME=$(uname)
+
+            if [ "$UNAME" == "Linux" ] ; then
```

Comment:
You can build the arm64 binary for the M1 on Linux too?

Reply:
yes. also amd64 darwin can be built in linux